### DISCUSSION TITLE
Unable to download Templates

### DISCUSSION THREAD

**Discussion by GabeSMG** [o3de/o3de#18110] [Help Wanted]
It's no longer possible to download the Multiplayer template.

```
[ERROR] o3de.download: SECURITY VIOLATION: Downloaded template.json contents do not match the advertised manifest json contents.
[ERROR] o3de.download: Could not validate zip, deleting
```

**Comment by spham-amzn** [comment]
It looks like there was an update to the template that was uploaded to o3de-extras releases without publishing the necessary updates to repo.json to update the hash. There is an open PR for https://github.com/o3de/o3de-extras/pull/735 so there needs to be another update anyways.

**Comment by AMZN-Gene** [comment]
https://github.com/o3de/o3de-extras/pull/735 doesn't update the repo.json
I've created a GHI https://github.com/o3de/o3de-extras/issues/739

**Comment by RezaSbu** [comment]
This issue is caused by a hash mismatch between the template package and the repo.json manifest.

The Multiplayer template was updated in **o3de-extras**, but the corresponding entry in **repo.json** was not updated with the new hash. When O3DE downloads the template, it validates the ZIP against the manifest hash — since they don’t match, the CLI reports:

"SECURITY VIOLATION: Downloaded template.json contents do not match the advertised manifest json contents."

This is why the validation fails and the template is deleted.

The maintainers already confirmed the root cause:
- The template in o3de-extras was updated
- The repo.json hash was NOT updated accordingly
- PR **o3de/o3de-extras#735** does not fix the repo.json entry
- A new GHI was opened (**o3de/o3de-extras#739**) to correct this

So the issue isn’t on your side — it will be resolved once the repo.json hash is updated and published in the extras release.

**Temporary workaround (until the hash is fixed):**
1. Clone the `o3de-extras` repo locally:
git clone https://github.com/o3de/o3de-extras.git

sql
Copy code
2. Register the local repo with O3DE:
o3de repo register --repo-path <path-to-local-o3de-extras>

pgsql
Copy code
3. Use the local template instead of downloading from the remote.

If this helps or clarifies the issue, feel free to mark this answer as accepted so others can find it eas
